project/src/components/city-card/city-card.tsx

```diff
@@ -40,7 +40,7 @@ function CityCard({offer, onHovered, onDeactivated, isActive}: CityCardType):JSX
       <div className='place-card__info'>
         <div className='place-card__price-wrapper'>
           <div className='place-card__price'>
-            <b className='place-card__price-value'>{offer.price}</b>
+            <b className='place-card__price-value'>€{offer.price}</b>
```

Comment:
старайся делать код симметричным. и так, чтобы в (недалеком) будущем новые требования заказчика можно было бы удовлетоврить с меньшими затратами
```suggestion
            <b className='place-card__price-value'>{formatPrice(offer)}</b>
```

```ts
const formatPrice = ({price}:{price:number})=>`€${price}`;
```